llpc/include/llpc.h

```diff
@@ -232,6 +232,30 @@ class ICompiler {
   /// @returns : Result::Success if successful. Other return codes indicate failure.
   virtual Result BuildShaderModule(const ShaderModuleBuildInfo *pShaderInfo, ShaderModuleBuildOut *pShaderOut) = 0;
 
+  /// Build unlinked shader to ElfPackage with part pipeline info.
+  ///
+  /// @param [in]  pPipelineInfo     : Info to build this shader module
```

Comment:
For historical reason, we keep old coding style for existing codes. But for newly added interfaces, please conform to current coding conventions: Don't use "p" prefix. Remove it in naming and comments.

Please go through this PR and close all pending review conversation if it is resolved. Thanks.

Reply:
Fixed. Thanks.